src/bootstrap/flags.rs

```diff
@@ -668,6 +677,11 @@ Arguments:
             rust_profile_generate: matches.opt_str("rust-profile-generate"),
             llvm_profile_use: matches.opt_str("llvm-profile-use"),
             llvm_profile_generate: matches.opt_present("llvm-profile-generate"),
+
+            allow: matches.opt_str("A"),
```

Comment:
This only lets you pass a single allow flag. We should let people pass multiple for consistency with clippy.

Reply:
What is the correct function to use to allow this?